docs/pages/database-access/guides/mysql-cloudsql.mdx

```diff
@@ -24,12 +24,38 @@ description: How to configure Teleport database access with GCP Cloud SQL MySQL.
   Service
 - (!docs/pages/includes/tctl.mdx!)
 
-## Step 1/5. Create a service account for the Teleport Database Service
+## Step 1/7. Enable Cloud SQL IAM authentication
 
-Teleport uses one-time passwords to authenticate with Cloud SQL MySQL. To be
-able to authenticate with a database instance, Teleport must run as a service
-account that has a few of the "Cloud SQL Admin" role permissions. You can create
-a new service account or modify an existing one to add required permissions.
+Teleport uses [IAM
+authentication](https://cloud.google.com/sql/docs/mysql/iam-authentication)
+with Cloud SQL MySQL instances.
+
+(!docs/pages/includes/database-access/cloudsql_enable_iam_auth.mdx type="MySQL" !)
+
+## Step 2/7. Create a service account to access the MySQL instance
```

Comment:
```suggestion
## Step 2/7. Create a service account for a database user
```

nit: I think it just makes it a little more clear that one svc acct is for db user, one is for teleport db service